supply-chain/catalog/trivy-scanning.hbs.md

```diff
@@ -0,0 +1,64 @@
+# Trivy Scanning Component
+
+## Description
+
+This document describes the Trivy Scanning Component for Tanzu Supply Chains. The Trivy Component provides out of the box trivy scanning that invokes [Supply Chain Security Tools - Scan 2.0](../../scst-scan/scan-2-0.hbs.md) in a Tanzu Supply Chain. See aquasecurity trivy [docs](https://github.com/aquasecurity/trivy) for more details on the trivy scanner.
```

Comment:
TODO: should we note that this is v0.48.3 as specified in lever?

Reply:
We should make it clear. Has it been set like this previously?

Reply:
According to our scanning [docs](https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.8/tap/scst-scan-integrate-app-scanning.html) for supplychain v1 we provided ClusterImageTemplates for Trivy/Grype but we didn't specify their versions.

I've been having some back and forth thoughts on this:
* The supply chains v1 and v2 provide a way to get started using what's available out of the box i.e.:
** supplychain v1 the ClusterImageTemplate Trivy/Grype are already provided and user can start trivy/grype scanning right away with the trivy/grype versions we provide
** supplchain v2 the Trivy Component is already provided and a user can start trivy scanning right away with the trivy version we provide
* However the supplychain v1 and v2 also allow the users to create/customize their own scanning experience:
** supplychain v1 users can create their own ClusterImageTemplate using whatever trivy/grype versions to scan with
** supplychain v2 users can create their own Component using whatever trivy version to scan with


Long story short:
* I feel like leaving this version specification out for now and letting this pass as a first draft. But I think I need to ask Ryan about this.